Formula/tea.rb

```diff
@@ -0,0 +1,23 @@
+class Tea < Formula
+  desc "Command-line tool to interact with Gitea servers"
+  homepage "https://gitea.com/gitea/tea"
+  url "https://gitea.com/gitea/tea/archive/v0.8.0.tar.gz"
+  sha256 "6c73c0a7b66cdfd1e5a302257d54df06a3a41eb9bdbfeb547966db431ae23b23"
+  license "MIT"
+  head "https://gitea.com/gitea/tea.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    # Remove `if build.head?` on next release.
+    system "make", "vendor" if build.head?
+    system "make"
+    bin.install "tea"
+    bash_completion.install "contrib/autocomplete.sh" => "tea"
+    zsh_completion.install "contrib/autocomplete.zsh" => "_tea"
```

Comment:
No fish completion?

Reply:
Probably
```ruby
fish_completion.install ".config/fish/conf.d/tea_completion.fish"
```

Reply:
Should I remove `.config/fish/conf.d/tea_completion.fish` after it's installed in the Fish completions directory, to not pollute ~/.config?

Reply:
On my machine the Fish completion file is installed by `tea` to `~/Library/Application Support/fish/conf.d/tea_completion.fish`, whch I really don't like (polluting my Fish directory I mean). What do you think about setting `XDG_CONFIG_HOME` to a directory in the current build directory?

Reply:
> Edit: wait, where is the `.config` in your comment? As in, which directory is it in?

The build directory. It's the same place the `contrib` directory is in.

> On my machine the Fish completion file is installed by `tea` to `~/Library/Application Support/fish/conf.d/tea_completion.fish`

That's surprising. The build sandbox shouldn't let you write to that location.

Reply:
> That's surprising. The build sandbox shouldn't let you write to that location. Or are you doing this outside of brew?

Yeah, I was wrong, not sure why I thought it wrote to my home directory. The line you suggested didn't work though, since the paths are different on macOS and Linux, and I had to prepend `.brew_home/` to the path.